src/record/export/utils/operateFilename.ts

```diff
@@ -0,0 +1,4 @@
+// https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file
+export const sanitizeFilename = (filename: string): string => {
```

Comment:
@ryounasso 

IMO: The name of the function is a bit generic, we can consider another specific name, for example: `replaceSpecialCharacters`

Reply:
I agree with your opinion.
So, fixed with d25a2e5d207680b506dee918a484c6d5ce03e0f9